libcipher/caesar.py

```diff
@@ -0,0 +1,64 @@
+__author__ = 'kskrishnasangeeth'
+
+import random
+import sys
+
+
+def encrypt(message, key=None):
+    """Returns the encrypted messsage for the string passed as argument.
+
+    Uses a random key for generating the cipher text
+
+    Args:
+        message (str) : The message to be encrypted.
+        key (int) : The key based on which encryption is performed. Is optional value.
+
+    Return:
+        str: the encrypted message """
```

Comment:
Move the triple quote to a new line. That is the pep8 recommendation.

    Unless the entire docstring fits on a line, place the closing quotes on a line by 
    themselves. This way, Emacs' fill-paragraph command can be used on it.